src/ci/docker/host-x86_64/x86_64-gnu-tools/Dockerfile

```diff
@@ -100,6 +100,5 @@ COPY scripts/shared.sh /scripts/
 # The `browser-ui-test.version` file is also used by bootstrap to emit warnings in case
 # the local version of the package is different than the one used by the CI.
 ENV SCRIPT /tmp/checktools.sh ../x.py && \
```

Comment:
You can now delete the comment above, it's outdated. The version file itself can also be deleted, and rustdoc-gui-test can just hardcode the version for now, later we'll switch it to `package.json`.

Btw the same outdated comment is also in `x86_64-gnu-miri/Dockerfile`, could you please also delete it from there? Thanks.